cattle/plugins/docker/compute.py

```diff
@@ -518,7 +519,9 @@ def _do_instance_activate(self, instance, host, progress):
                                          START_CONFIG_FIELDS)
 
         add_label(create_config, {UUID_LABEL: instance.uuid})
-        add_label(create_config, {'io.rancher.container.name': instance.name})
```

Comment:
What should the behavior be for this label if the rancher container didnt have a name? I went with _not_ setting the name label.